spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -1,12 +1,34 @@
+
+=== Provider Name
+
+The entity annotation class is the primary strategy to avoid conflicts between Jakarta Data providers. In most cases, it is sufficient to differentiate between providers. In situations where multiple Jakarta Data providers support the same entity annotation class, the application can specify the name of the desired Jakarta Data provider using the optional `provider` attribute of the `Repository` annotation.
+
+To ensure compatibility and prevent conflicts, Jakarta Data providers must disregard all `Repository` annotations that specify a different provider's name through the `Repository.provider()` attribute. By ignoring these annotations, Jakarta Data providers allow other Jakarta Data providers to handle them.
+
 == Interoperability with other Jakarta EE Specifications
 
-In this section, we will delve into the robust capabilities of Jakarta EE Data and its seamless integration with other Jakarta EE specifications. This integration offers a comprehensive data persistence and management solution in Java applications. When operating within a Jakarta EE product, the availability of other Jakarta EE Technologies depends on the profile. This section outlines how related technologies from other Jakarta EE Specifications work together with Jakarta Data to achieve interoperability.
+This section discusses interoperability with related Jakarta EE specifications. When operating within a Jakarta EE product, the availability of other Jakarta EE technologies depends on whether the Jakarta EE Core profile, Jakarta EE Web profile, or Jakarta EE Platform is used.
 
 === Jakarta Contexts and Dependency Injection
 
-Contexts and Dependency Injection (CDI) is a core specification in Jakarta EE that provides a powerful and flexible dependency injection framework for Java applications. CDI lets you decouple components and manage their lifecycle through dependency injection, enabling loose coupling and promoting modular and reusable code.
+Contexts and Dependency Injection (CDI) is a specification in the Jakarta EE Core profile that provides a powerful and flexible dependency injection framework for Java applications. CDI lets you decouple components and manage their lifecycle through dependency injection, enabling loose coupling and promoting modular and reusable code.
 
-One of the critical aspects of CDI integration with Jakarta EE Data is the ability to create repository instances using the `@Inject` annotation effortlessly. Repositories are interfaces that define data access and manipulation operations for a specific domain entity. Let's take an example to illustrate this integration:
+In Jakarta EE products, CDI provides the ability to register implementation of Jakarta Data repositories for injection into applications via the the `@Inject` annotation. Repositories are interfaces that define data access and manipulation operations for a specific domain entity. The following example illustrates this integration:
```

Comment:
```suggestion
In Jakarta EE products, CDI provides the ability to register implementations of Jakarta Data repositories for injection into applications via the the `@Inject` annotation.  The following example illustrates this integration:
```
This doesn't seem like the right point to define Repositories

Reply:
I agree. It seemed out of place to me as well.  I'll add your suggestion to remove it.